examples/server.js

```diff
@@ -102,7 +102,7 @@ app.use('/api/speech-to-text/token', function(req, res) {
       res.status(500).send('Error retrieving token');
       return;
     }
-    res.send(token);
+    res.send(token.token);
```

Comment:
Really? I believe you that it works but I am confused as to why... If the service is RC, this should return a string

Reply:
if you are right then we have a problem because it should return the same independently of if is a cf or RC. I was testing with CF credentials so maybe my change is only needed for CF. I can add a fall back like
```js
token.token || token
```

Reply:
I think that line is a good temporary solution while we figure out what's going on